charts/federation-v2/README.md

```diff
@@ -101,11 +101,8 @@ chart and their default values.
 | controllermanager.image               | Name of the federation v2 image.                                                                                                                                                                            | federation-v2                                                                                         |
 | controllermanager.tag                 | Tag of the federation v2 image.                                                                                                                                                                             | latest                                                                                                |
 | controllermanager.pullPolicy          | Image pull policy.                                                                                                                                                                                          | IfNotPresent                                                                                          |
-| controllermanager.featureGates        | Feature gates are a set of `key=value` pairs that describe alpha or experimental features. An administrator can use the `--feature-gates` command line flag on each component to turn a feature on or off.  | PushReconciler=true,SchedulerPreferences=true,CrossClusterServiceDiscovery=true,FederatedIngress=true |
-| controllermanager.federationNamespace | The namespace the federation control plane is deployed in.                                                                                                                                                  | federation-system                                                                                     |
```

Comment:
I'm just noticing that the federation namespace can be specified as a helm argument.  Why would that ever be done vs using the downward API to determine the namespace the deployment is running in?

Reply:
It is removed now. And I use downward API to set the hosted namespace in environment variables of helm definition and get in controller during startup.